Fix:
error: implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
--- lastfmlib/utils/condition.cpp.orig	2009-07-03 10:26:53.000000000 -0500
+++ lastfmlib/utils/condition.cpp	2021-12-28 22:28:22.000000000 -0600
@@ -19,6 +19,7 @@
 #include "mutex.h"
 #include <sys/time.h>
 #include <errno.h>
+#include <string>
 #include <string.h>
 #include <stdexcept>
 
